### DISCUSSION TITLE
Probably require node's querystring module without node: prefix?

### DISCUSSION THREAD

**Discussion by BobbyWibowo** [kartikk221/hyper-express#114] [Q&A] [answered]
The ability to require node's built-in modules with `node:` prefix was added in Node 16 release, and was only later backported to Node 14 starting with v14.18.0 ([changelog](https://github.com/nodejs/node/releases/tag/v14.18.0))

And I think, unless hyper-express explicitly require other features that were only backported in later Node 14 versions, uWebSockets.js should work throughout the entirety of the Node 14 version range
I have no proof of that statement however, since uWebSockets.js `binaries` branch's `package.json` does not have engine versions requirements, but its most recent GitHub release  which explicitly mentions Node versions ([Release v20.7.0](https://github.com/uNetworking/uWebSockets.js/releases/tag/v20.7.0)), did not mention specific version range for Node 14, so..

And at least by hyper-express@6.4.2, my simple upload test project ([hyper-express-playground](https://github.com/BobbyWibowo/hyper-express-playground )) seemingly work fine with flat Node 14.0.0

Not to say I condone not updating your Node installation, since security patches and all of course matter, but at the same time probably not worth increasing the version requirement if not really necessary (?)

**Comment by kartikk221** [comment, marked as answer]
Yeah, there doesn't seem to be any harm in removing the use of the `node:` prefix since there won't be any package name collisons and should automatically expand support across all versions of Node 14. Will be pushed in next update.

**Reply by kartikk221** [reply]
This change has been pushed in the v6.4.7 release.
